src/Mod/PartDesign/App/FeatureHelix.cpp

```diff
@@ -77,21 +77,22 @@ Helix::Helix()
 {
     addSubType = FeatureAddSub::Additive;
 
-    ADD_PROPERTY_TYPE(Base, (Base::Vector3d(0.0, 0.0, 0.0)), "Helix", App::Prop_ReadOnly, "Base");
-    ADD_PROPERTY_TYPE(Axis, (Base::Vector3d(0.0, 1.0, 0.0)), "Helix", App::Prop_ReadOnly, "Axis");
-    ADD_PROPERTY_TYPE(Pitch, (10.), "Helix", App::Prop_None, "Pitch");
-    ADD_PROPERTY_TYPE(Height, (30.0), "Helix", App::Prop_None, "Height");
-    ADD_PROPERTY_TYPE(Turns, (3.0), "Helix", App::Prop_None, "Turns");
+    const char* group = "Helix";
+    ADD_PROPERTY_TYPE(Base, (Base::Vector3d(0.0, 0.0, 0.0)), group, App::Prop_ReadOnly, "The center point of the helix' start.");
+    ADD_PROPERTY_TYPE(Axis, (Base::Vector3d(0.0, 1.0, 0.0)), group, App::Prop_ReadOnly, "The helix' direction.");
```

Comment:
No, unfortunately not. What you mean is the *property* `ReferenceAxis`, which is called *in the UI of the task panel* just "Axis". The *property* `Axis` is only the direction. It is derived from `ReferenceAxis` (just like `Base`) and because of this read-only.

I know, it's a complete mess, but unfortunately we cannot change the property names without breaking existing models (at least AFAIK).

Reply:
Aha. So then the tooltip for "Reference axis" should reflect this.